docs/conf.py

```diff
@@ -57,9 +57,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.2.2'
+version = '1.4.0rc1'
```

Comment:
Similar to the comment below, this `conf.py` file is configuration for the documentation built to https://docs.securedrop.org/en/master/ . I believe it's expected to describe the current release, not the next one. See the changes to `conf.py` in a previous backport: https://github.com/freedomofpress/securedrop/pull/5061 for comparison